subprojects/extTest/src/test/java/org/mockitousage/plugins/stacktrace/MyStackTraceCleanerProvider.java

```diff
@@ -13,11 +13,11 @@
     public StackTraceCleaner getStackTraceCleaner(final StackTraceCleaner defaultCleaner) {
         return new StackTraceCleaner() {
 			@Override
-			public boolean isOut(StackTraceElement candidate) {
+			public boolean isIn(StackTraceElement candidate) {
 				if (ENABLED && candidate.getMethodName().contains("excludeMe")) {
                     return false;
```

Comment:
It seems impossible that this could have ever worked before. The test wanted an exclusion of "excludeMe" functions when ENABLED was true. `isOut()` should return true to exclude something in the original. This would return false in the case that was being tested. It was hard to understand what this was getting at when you have to return true to say whether something is NOT included in a list.

Unless I've totally misunderstood this, then the refactoring and modification should be on the side of making things better. What I can't understand is why this isn't failing the main build!